<details>
<summary>Pre-Commit Test failed! Click here</summary>

```
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8:pep8-naming.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Tests should end in _test.py.........................(no files to check)Skipped
Check for added large files..............................................Passed
Check for byte-order marker..............................................Passed
Check for case conflicts.................................................Passed
Check docstring is first.................................................Passed
Check JSON...............................................................Passed
Check for broken symlinks............................(no files to check)Skipped
Detect Private Key.......................................................Passed
Check python ast.........................................................Passed
Debug Statements (Python)................................................Passed
pydocstyle...............................................................Passed
Check Toml...............................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

producer.py:185: error: Name 'indexes' is not defined
Found 1 error in 1 file (checked 2 source files)

black....................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

producer.py:185:72: F821 undefined name 'indexes'
```
</details>

--- a/producer.py
+++ b/producer.py
@@ -163,6 +163,7 @@ async def main() -> None:
 
     # We dont fetch unsolved packages if both solver and revsolver messages are disabled.
     if THOTH_GRAPH_REFRESH_SOLVER or THOTH_GRAPH_REFRESH_REVSOLVER:
+        indexes = _GRAPH_DB.get_python_package_index_urls_all()
         packages = _unsolved_packages(packages=packages)
 
     if not packages:
@@ -270,19 +271,19 @@ async def main() -> None:
                 )
 
     _METRIC_MESSSAGES_SENT.labels(
-        message_type=UnresolvedPackageMessage.name,
+        message_type=UnresolvedPackageMessage.topic_name,
         env=THOTH_MY_NAMESPACE,
         version=__service_version__,
     ).inc(solver_messages_sent)
 
     _METRIC_MESSSAGES_SENT.labels(
-        message_type=UnrevsolvedPackageMessage.name,
+        message_type=UnrevsolvedPackageMessage.topic_name,
         env=THOTH_MY_NAMESPACE,
         version=__service_version__,
     ).inc(revsolver_messages_sent)
 
     _METRIC_MESSSAGES_SENT.labels(
-        message_type=SIUnanalyzedPackageMessage.name,
+        message_type=SIUnanalyzedPackageMessage.topic_name,
         env=THOTH_MY_NAMESPACE,
         version=__service_version__,
     ).inc(security_messages_sent)

